src/components/Table/components/TableHeader/TableHeader.js

```diff
@@ -138,7 +138,7 @@ const condensedColStyles = ({ condensed, scope, theme }) =>
   `;
 
 const StyledHeader = styled('th', {
-  shouldForwardProp: (prop) => isPropValid(prop) && prop !== 'scope',
+  shouldForwardProp: (prop) => isPropValid(prop),
```

Comment:
@connor-baer this whole PR is still a 🚧 but just FYI, since we talked about it, this is what prevented the `scope` (which we were correctly setting) to make it to the DOM 🙂

Reply:
Huh, why did we omit that? Weird. Thanks for fixing it!

Reply:
Oh no, it was me 😳 I misunderstood what the prop was doing 😬 TIL

Reply:
Yes because we're using the same prop to handle styling logic - it's a bit confusing 🙂